apps/mainsite/management/commands/reminders_direct_awards.py

```diff
@@ -36,8 +37,8 @@ def handle(self, *args, **kwargs):
         index = len(threshold_days) - 1
         unaccepted = 'Unaccepted'
         for days in threshold_days:
-            reminder_date = now - datetime.timedelta(days=days)
-            direct_awards = DirectAward.objects.filter(created_at__lt=reminder_date,
+            reminder_cutoff = now + timedelta(days=days)
+            direct_awards = DirectAward.objects.filter(expiration_date__lt=reminder_cutoff,
                                                        reminders=index,
                                                        expiration_date__isnull=False,
```

Comment:
I think this row can be removed?

Reply:
Which row?

Reply:
expiration_date__isnull=False,

As we now filter on expiry date anyway.